querybook/webapp/ui/SearchBar/SearchBar.tsx

```diff
@@ -46,7 +46,7 @@ export const SearchBar: React.FunctionComponent<ISearchBarProps> = ({
             // setTimeout is needed to make the focus work
             // otherwise the component
             setTimeout(() => {
-                inputRef.current.focus();
+                inputRef?.current?.focus();
```

Comment:
inputRef.current can be null?
in this case, inputRef shouldn't be null, can you change it to `inputRef.current?.focus()`